docs/source/projects.rst

```diff
@@ -87,6 +88,28 @@ Conda environment
   file in the root of the project directory or by including a ``conda_env`` entry in your
   ``MLproject`` file. For details, see the :ref:`project-directories` and :ref:`mlproject-specify-environment` sections.
 
+  The ``mlflow run`` command supports running a Conda environment project as a Python environment project.
+  To do this, run ``mlflow run`` with ``--env-manager local``:
+
+  .. code-block:: bash
+
+      mlflow run /path/to/conda/project --env-manager virtualenv
+
+  .. warning::
+
+      When a Conda environment project is executed as a Python environment project,
+      conda dependencies will be ignored and only pip dependencies will be installed.
+
+.. _project-python-environments:
+
+Python environment
+  Python environments support Python packages available on PyPI. When an MLflow Project
+  specifies a Python environment, MLflow creates it using ``virtualenv`` and ``pyenv``, and
```

Comment:
Do we want to say what these elements are doing? 
"MLflow will download the specified version of Python by using ``pyenv`` and create a ``virtualenv`` that contains the exact package dependencies and versions that were used during training, activating it as the execution environment prior to running the project code."
(or something like that)

Reply:
Sounds good! I'll add it.

Reply:
How about

> MLflow will download the specified version of Python by using pyenv and create an isolated environment that contains the project dependencies using virtualenv, activating it as the execution environment prior to running the project code.

?

Reply:
sounds good!